terminusdb_client/woqlclient/woqlClient.py

```diff
@@ -72,7 +75,7 @@ def connect(self, **kwargs):
         self.conCapabilities.set_capabilities(json_obj)
         return json_obj
 
-    def copy(self):
+    def copy(self) -> WOQLClient:
```

Comment:
I think it needs to be "WOQLClient" in the WOQLClient" class itself. Otherwise, there will be an error (I think it is casing the Travis to fail)

Reply:
Good point, I missed that. Fixed now.